app/services/api/declarations/create.rb

```diff
@@ -0,0 +1,142 @@
+module API::Declarations
+  class Create
+    include API::Concerns::Teachers::SharedAction
+
+    attribute :declaration_date
+    attribute :declaration_type
+    attribute :evidence_type
+
+    validates :declaration_date, presence: { message: "Enter a '#/declaration_date'." }
+    validates :declaration_date,
+              future_date: true,
```

Comment:
This validator has caught me out in ECF before - it looks like the declaration date has to be future dated, but it actually needs to be in the past. Can we just inline this validation into the service? I think unless we expect to reuse the validation elsewhere or its suitably complicated (like the evidence held validator) then I would opt to just inline them in the service to make it easier to follow